rel-eng/atomic-enterprise-allinone.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=Atomic Enterprise Master All-In-One
```

Comment:
So what is the plan around breaking out the apiserver? Breaking out and load-balancing the apiserver is the simplest thing todo to increase performance and scalability.  

cc @jeremyeder @smarterclayton